### ISSUE TITLE
MCQ fails when an item is selected

### ISSUE THREAD

**Issue by nachocinalli** [adaptlearning/adapt-contrib-core#353]
### MCQ fails when an item is selected
Get an Uncaught DOMException: Failed to execute 'querySelector' error in file core/js/fixes/safari.label.click.blur.js at line 26

### Your environment
* Adapt Framework 5.28.15
* Chrome
* Windows

### Steps to reproduce
1. Create a MCQ
2. Select an item

### Expected behaviour
The item should be selected

### Actual behaviour
The item is not selected and the error is thrown
